packages/frontend/src/creation-form/index.tsx

```diff
@@ -67,6 +59,22 @@ export const Component = ({
         state.openingTimestamp ? dayjs.unix(state.openingTimestamp) : null
     );
     const [minimumBond, setMinimumBond] = useState(state.minimumBond || "");
+    const [questionErrorText, setQuestionErrorText] = useState("");
+    const [questionTimeoutErrorText, setQuestionTimeoutErrorText] =
+        useState("");
+    const [minimumBondErrorText, setMinimumBondErrorText] = useState("");
+
+    const [minimumDate, setMinimumDate] = useState(new Date());
+
+    useEffect(() => {
+        const interval = setInterval(() => {
```

Comment:
Why do we need an `interval` here?

Reply:
It's used to update the minimum date every second. It has the nice effect of updating the date picker in realtime with the values you can or cannot pick